02-multiple-target-environments/readme.md

```diff
@@ -117,3 +117,70 @@ git push
 ```
 
 Now if we check the workflow we will see three jobs each one refering a different node version.
+
+## Using a multi-dimension matrix
+
+In this case we want to execute our job in two different OS, Ubuntu 20.04 and Ubuntu 22.04. We can specify a second variable in our matrix to create a multi-dimensional one. A job will run for each possible combination of the variables.
+
+```diff
+jobs:
+  build-test:
+-   runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x, 16.x]
```

Comment:
Maybe we can upgrade the Node versions to 16, 17 and 18

Reply:
Updated